frontend/src/metabase/public/components/EmbedModal/EmbedModalContent/EmbedModalContent.tsx

```diff
@@ -20,6 +25,15 @@ import type {
 import { SelectEmbedTypePane } from "../SelectEmbedTypePane";
 import { EmbedModalContentStatusBar } from "./EmbedModalContentStatusBar";
 
+const countEmbeddingParameterOptions = (embeddingParams: EmbeddingParameters) =>
```

Comment:
We want to count the number of parameters for each embed state for the snowplow event i.e. disabled has 1, locked has 0, etc